plugin/pulp/plugin/serializers.py

```diff
@@ -0,0 +1,2 @@
+# All Serializers in platform are potentially useful to plugin writers
```

Comment:
It is not clear in pulp.app.serializer.__init__.py that everything imported there will be part of the plugin API. I think we should add a note there or explicitly import each serializer.